frontends/ol-components/src/components/Link/Link.tsx

```diff
@@ -33,16 +32,22 @@ const NO_FORWARD = Object.keys({
  */
 const linkStyles = (props: LinkStyleProps) => {
```

Comment:
IMO it's a bit weird that the typography style depends on color, but that's what how our [designs](https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=470-11729&m=dev) do it, so worth matching that rather than applying the bolder black font on a case-by-case basis.